### ISSUE TITLE
Unique Page Titles

### ISSUE THREAD

**Issue by zebdelk** [cedardevs/onestop#514]
We're currently missing WCAG 2.0 2.4.2, which requires providing page titles for navigation purposes.

We should have unique page titles for each navigation route, but currently the page title is completely static.

- Home
- About
- Help
- Collection Search Results
- Collection Details
  - should change for each ID
- Granule Search Results

Each title should front load the information that is different, and end with NOAA OneStop.

For the collection details, the collection titles are too long to put in web page titles, so we should consider what other reasonably unique options we have or if this is a good place to use truncated titles. (The IDs are unique, but also not human readable, so it's debatable if that's better or worse). For example, Amazon product pages title with the name of the item, such as "[album name] by [artist] on Amazon Music - Amazon.com"

Considerations for implementation:

We should probably upgrade our router first, so that we don't add more tech debt to that by our choice of solution for updating the page title.

Then we should pick a solution for updating the title dynamically based on the redux store with the routing path. (See below for one possibility).

For more info:

https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-title.html

https://github.com/ReactTraining/react-router/issues/49#issuecomment-58427043

Note for priority: This is considered level A, which is the most basic functionality in WCAG.

**Comment by zebdelk** [comment]
Related to #593

**Comment by zebdelk** [comment]
- [ ] update to react router 4
- [ ] use helmet to set page titles, and other simple metadata
